target/linux/generic/pending-4.14/930-gpio-add-support-for-apu2-apu3-devices.patch

```diff
@@ -0,0 +1,395 @@
+--- /dev/null
```

Comment:
why is this inside generic and not x86 ?

Reply:
Yes you are right this must go to
> target/linux/x86/patches-4.14/